tests/workflows/theiacov/test_wf_theiacov_fasta.yml

```diff
@@ -4,12 +4,11 @@
     - wf_theiacov_fasta
     - wf_theiacov_fasta_cromwell
   files:
-    - path: log.err
-      contains: ["Successfully released change log lock", "INFO", "WORKFLOW_METADATA_SUMMARY_ENTRY", "liquibase"]
+    - path: log.err # file is empty, so don't check for strings within, just for presence
     - path: log.out
       contains: ["workflow finished with status 'Succeeded'", "theiacov_fasta", "Done"]
     - path: metadata.json
-      contains: ["outputs", "theiacov_fasta", "Succeeded"]
+      contains: ["outputs", "theiacov_fasta", "status", "Succeeded"]
```

Comment:
is there a reason why you added "status" here?

Reply:
I was messing around and trying to get a complete string that included double quotes `"status:" "Succeeded"`, but it got too complicated so I just added this check for the word `status`. Not necessary but it is present in the `metadata.json` file

Reply:
I think I'd prefer it be present in all or none, do you agree?

Reply:
yes, I'll change it now so that it's not in any of the ymls 👍